ldap_probe/models.py

```diff
@@ -51,7 +50,7 @@ class LdapProbeLogFullBindManager(models.Manager):
     will allow full LDAP binds for a limited number of `AD` controllers.
     By design LDAP probe data collected from these `AD` controllers will
     present with :attr:`LdapProbeLog.elapsed_bind` values that are not
-    0.
+    null (`None` in `Python` speak).
```

Comment:
None could be in double backticks, so it renders as a code literal.
```suggestion
    null (``None`` in `Python` speak).
```

Reply:
That would go against the convention used everywhere else in the docstrings. All True/False/None entries are single back-ticked

Reply:
This is not true in general, see for example p_soc_auto_base/admin.py

Is this the intended convention?